lib/cypress/component/index.spec.jsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import Test from "../../testing-components/Test";
+import TestUseStoryblok from "../../testing-components/TestUseStoryblok";
+import { mount } from "@cypress/react";
+import Teaser from "gatsby-source-storyblok-playground/components/teaser";
+import Grid from "gatsby-source-storyblok-playground/components/grid";
+import Feature from "gatsby-source-storyblok-playground/components/feature";
+import Page from "gatsby-source-storyblok-playground/components/page";
+
+describe("gatsby-source-storyblok", () => {
+  beforeEach(() => {
+    cy.spy(window.console, "log").as("log");
+    cy.spy(window.console, "error").as("error");
+    delete window.storyblokRegisterEvent;
+    document.getElementById("storyblok-javascript-bridge")?.remove();
+  });
+
+  describe("JS Bridge", () => {
```

Comment:
Question about the tests:

I see that the only function created in `lib/index.js` is `useStoryblokState`, but in the file `lib/cypress/component/index.spec.jsx` I see tests related to "useStoryblok", the StoryblokComponent, and the JS Bridge.

Are these tests different from the ones we already have in `@storyblok/react`?
https://github.com/storyblok/storyblok-react/blob/feature/v1/lib/cypress/component/index.spec.jsx

Reply:
Good catch! Also, in Gatsby SDK, it's `useSToryblokState` instead of `useStoryblok`.

 @alexjoverm and I had a conversation that we don't need to test common APIs that are coming from React SDK, which means that we don't need to duplicate the test.

I'll fix it to make sense.

Reply:
👍